helsingborg-stad/app-mitt-helsingborg
# fix(cocoapods): set missing team id for some pods

workaround for a bug that might be fixed in later xcode versions.

I think I took the solution from [here](https://github.com/CocoaPods/CocoaPods/issues/11402#issuecomment-1256113429).

## How to test

Concrete example:

1. Checkout this branch
2. Verify `yarn ios` etc. work as usual
3. If you get "X requires a development team" run `pod install` and try again.

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.
